CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing
```

Comment:
```suggestion
# Contribution Guide
```

Reply:
I am also drafting a new CONTRIBUTING.md recently for the docs repo. I may add more contribution guidelines to this file later and reorganize them a little. Anyway, thanks for your first draft of this! :) @gregwebs